### PR TITLE
#161967118 Integrate Travis CI

### PR DESCRIPTION
**Pull request by Darius-Ndubi**
#### What does this PR do?
    Configures the application for testing.
----------

#### Description of Task to be completed?
  - Integrates Travis with Authors Haven Django app.
  - Adds a badge for the builds on the readme.
------------

#### What are the relevant pivotal tracker stories?
  - [#161967118](https://www.pivotaltracker.com/story/show/161967118)
--------------

#### Checklist
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings

### CODE REVIEW

**Comment by jillo-abdullahi** [comment]
@Darius-Ndubi Are the checklists under **Type of Change** from the template supposed to act as a guide or should they also be on the actual PR? @mutoromike

**Comment by Darius-Ndubi** [comment]
I am working to create a template on the PR format to integrate the two and ask if its okay to use it.

**Comment by bettblake08** [comment]
@darius I believe all is good with the PR. The structure of the PR still needs to be improved on to meet the Andela conventions.

**Comment by mutoromike** [comment]
I think the PR is okay, and also the commits match the engineering playbook requirements. For PRs, we will be using the PR format suggested by @evansmusomi, this was previously shared.

**Review by mutoromike — CHANGES_REQUESTED** [review]
The Travis Badge is not working, even on your branch. Kindly fix it.

**Review by bettblake08 — COMMENTED** [review]
The PR is all good.

**Comment by Darius-Ndubi** [comment]
@mutoromike the conflicts have been solved, thank you for the feedback.

File: .coveralls.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+repo_token: BQg4BzLvHTeY9Rb96xVCSHqY2iEFoJatm
```
Comment (Bryan-Cee) [inline .coveralls.yml]: Please add a new line to this file to act as the EOL.

Comment (Bryan-Cee) [inline .coveralls.yml]: Please add a new line at the end of this file as the EOL.

**Review by Bryan-Cee — CHANGES_REQUESTED** [review]

**Comment by davishooly** [comment]
@Darius-Ndubi concerning the coveralls badge its still pointing to master branch it should point to develop

**Review by mutoromike — APPROVED** [review]

**Pull request closed without merging**